msteams-platform/concepts/app-fundamentals-overview.md

```diff
@@ -48,7 +48,7 @@ Understanding the user and their concern are the first indicators of how a Teams
 
 * **Use Teams as a platform**: Teams platform helps you build a single- or multi-capability app. Your Teams app is supported by the integrated products and services that strengthen the app experience.
 
-    :::image type="content" source="../assets/images/overview/teams-solution.png" alt-text="Conceptual representation of the Teams solution.":::
+    :::image type="content" source="../assets/images/overview/teams-solution.png" alt-text="The screenshot is an example that shows a conceptual representation of the Teams solution.":::
```

Comment:
* This image is not a screenshot of any UI.
* The infographic is not an example of anything.
* It is not a representation of the Teams solution.
* Overall the alt-text doesn't convey what the image is about.

CC @v-habibaumme , @v-psambrani .

Reply:
changed the screenshot to image